go/fn/object_test.go

```diff
@@ -10,6 +10,78 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type GVKTest struct {
+	group   string
+	version string
+	kind    string
+	resultA bool
+	resultB bool
+	resultC bool
```

Comment:
nit: give these results more descriptive names. I'm not really sure what they are supposed to be